contributing/setting-up-the-project.md

```diff
@@ -68,9 +68,13 @@ Currently the install command is also what builds all of the individual packages
 ## Create certificates
 
 ```sh
-pnpx @warp-drive/holodeck ensure-cert
+pnpm dlx @warp-drive/holodeck ensure-cert
 ```
 
+## Building the project
+
+The project's packages will build whenever `pnpm install` is run.
+
```

Comment:
I'm intending to do a full rundown of "how to build, run and rebuild all the things" here but first I'm going to simplify the scripts to do so